flink-end-to-end-tests/test-scripts/common.sh

```diff
@@ -729,21 +729,29 @@ function find_latest_completed_checkpoint {
 }
 
 function retry_times() {
+    retry_times_with_backoff_and_cleanup $1 $2 "${@:3}" "true"
```

Comment:
How well does it mitigate the issue? Did you run the test with multiple iterations?